### PR TITLE
Effect panel shows effect descriptions and can send effects to chat

### PR DESCRIPTION
**Pull request by JulieWinchester**
This PR addresses feature request https://github.com/foundryvtt/pf2e/issues/4316. Specifically, it:

- Modifies the Effect Panel so that afflictions, conditions, and effects show effect descriptions in the hovering effect information. The description can be scrolled in its own UI element if the effect description is overly long. The description text can display enriched document links, which can be interacted with. 
- Adds an icon to the right of the effect name in the hovering effect information that allows the full effect description to be sent to chat. The send to chat icon uses the same hover indication as the effect info persistent damage recovery roll link (orange text-shadow).

I've added screenshots below that demonstrate the changes. This PR does potentially overlap with https://github.com/foundryvtt/pf2e/pull/4527. Sorry for that, I didn't see that PR until after I made these changes (primarily to familiarize myself with the system). I'll of course be happy to close this PR if desired due to that.

For any questions or more information, I'm on the discord as JellyfishJail#3956. 

Screen Shot 2023-03-18 at 9 26 24 PM

Screen Shot 2023-03-18 at 9 25 58 PM

Screen Shot 2023-03-18 at 9 26 10 PM

### CODE REVIEW

**Comment by stwlam** [comment]
Nice work: I think we'd prefer this one.

**Comment by CarlosFdez** [comment]
Very neat!

File: src/module/apps/effects-panel.ts [hunk 1]
```diff
@@ -33,7 +34,14 @@ export class EffectsPanel extends Application {
     override async getData(options?: ApplicationOptions): Promise<EffectsPanelData> {
         const { actor } = this;
         if (!actor || !game.user.settings.showEffectPanel) {
-            return { afflictions: [], conditions: [], effects: [], actor: null, user: { isGM: false } };
+            return {
+                afflictions: [],
+                conditions: [],
+                effects: [],
+                descriptions: {} as EffectsDescriptionData,
```
Comment (stwlam) [inline src/module/apps/effects-panel.ts:41]:
```suggestion
                descriptions: { afflictions: [], conditions: [], effects: [] },
```

**Comment by stwlam** [comment]
Closes #4527

**Pull request merged**